ethereum/blockcreation/src/test/java/org/hyperledger/besu/ethereum/blockcreation/PoWBlockCreatorTest.java

```diff
@@ -52,9 +52,9 @@
 
 import com.google.common.collect.Lists;
 import org.apache.tuweni.bytes.Bytes;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-public class PoWBlockCreatorTest {
+public class PoWBlockCreatorTest extends AbstractBlockCreatorTest {
```

Comment:
If AbstractBlockCreatorTest isn't abstract, do we need to extend it here?

Reply:
Hmm.. good point mostly wanted the name to match the test class as I put common logic in AbstractBlockCreator. Rename it to BaseBlockCreatorTest maybe?

Reply:
Just thinking that PoW will rerun AbstractBlockCreatorTest's tests as well as the parent I think - is that the intention?

Potentially could make AbstractBlockCreatorTest abstract so they run for subclasses but not the parent